ansible/www-standalone/tools/promote/upload_to_cloudflare.sh

```diff
@@ -42,6 +42,10 @@ fi
 relativedir=${dstdir/$dist_rootdir/"$site/"}
 tmpversion=$2
 
-aws s3 cp $staging_bucket/$relativedir/$tmpversion/ $dist_bucket/$relativedir/$tmpversion/ --endpoint-url=$cloudflare_endpoint --profile $cloudflare_profile --recursive --no-follow-symlinks
+# Due to R2 limitations, `aws s3 cp` and `aws s3 sync` only succeed in copying
+# different sets of files across. Fortunately the sets are disjoint, so running
+# both commands (ignoring errors) will copy all the files across.
```

Comment:
Out of scope of this PR, but I wonder if it would be worth switching to rclone for moving files around like this? I've found it to be reliable across different S3 providers (rclone has various workarounds for different S3 providers.)